CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## Version 0.18.0
+
+### Summary
+
+This release cycle focused on review and approval of two RFCs, one for Readout Error Mitigation (REM) [#1387](https://github.com/unitaryfund/mitiq/issues/1387) and one for Identity insertion noise scaling [#335](https://github.com/unitaryfund/mitiq/issues/335) (not listed as PRs). It also includes bug fixes and minor dependency upgrades. 
+
+### All Changes
+
+- Update black requirement from ~=22.6 to ~=22.8  infrastructureFor issues related to building, packaging, and continuous integration.
+#1477 by [@dependabot] 
```

Comment:
```suggestion
### All Changes

- Update amazon-braket-sdk requirement from ~=1.29.2 to ~=1.29.3 . #1481 by [@dependabot] 

- Update black requirement from ~=22.6 to ~=22.8  infrastructureFor issues related to building, packaging, and continuous integration.
#1477 by [@dependabot] 
```
I merged a new dependabot PR (I should have probably waited), so we need new line here.